src/domain/application/applicationView/hooks/useSendMuutosilmoitus.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Button, IconEnvelope, Notification } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+import { AlluStatus, Application } from '../../types/application';
+import { validationSchema as johtoselvitysValidationSchema } from '../../../johtoselvitysTaydennys/validationSchema';
+import { validationSchema as kaivuilmoitusValidationSchema } from '../../../kaivuilmoitusTaydennys/validationSchema';
+import { isContactIn } from '../../utils';
+import { SignedInUser } from '../../../hanke/hankeUsers/hankeUser';
+import ApplicationSendDialog from '../../components/ApplicationSendDialog';
+
+const validationSchemas = {
+  CABLE_REPORT: johtoselvitysValidationSchema,
+  EXCAVATION_NOTIFICATION: kaivuilmoitusValidationSchema,
+};
+
+export default function useSendMuutosilmoitus(
+  application: Application,
+  signedInUser: SignedInUser | undefined,
+) {
+  const { t } = useTranslation();
+  const validationSchema = validationSchemas[application.applicationType];
+  const isValid = validationSchema?.isValidSync(application.muutosilmoitus);
+  const showSendButton =
+    application.alluStatus === AlluStatus.DECISION &&
```

Comment:
Should send button be shown also when Allu status is `OPERATIONAL_CONDITION` as muutosilmoitus can be done in that case too?

Reply:
You're right! Added check for that status as well.